src/API/src/export/export.controller.ts

```diff
@@ -0,0 +1,10 @@
+import { Controller, Get, Res, StreamableFile } from '@nestjs/common';
+
+@Controller('export')
+export class ExportController {
+  @Get('download-all')
```

Comment:
Can this be changed to `downloadAll`? We've used camel casing for other endpoints.